pilot/pkg/networking/core/v1alpha3/route/retry/retry_test.go

```diff
@@ -30,203 +30,200 @@ import (
 	"istio.io/istio/pilot/pkg/networking/util"
 )
 
-func TestNilRetryShouldReturnDefault(t *testing.T) {
-	g := NewWithT(t)
-
-	// Create a route where no retry policy has been explicitly set.
-	route := networking.HTTPRoute{}
-
-	policy := retry.ConvertPolicy(route.Retries)
-	g.Expect(policy).To(Not(BeNil()))
-	g.Expect(policy).To(Equal(retry.DefaultPolicy()))
+type testCase struct {
```

Comment:
You do not need a separate struct you can define an anonymous struct like this https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go#L79